lib/icalendar_helper.rb

```diff
@@ -0,0 +1,40 @@
+require 'icalendar' # https://github.com/icalendar/icalendar
+
+# Includes all abstractions over iCalender representations
+module IcalendarHelper
+  # Converts a sticky activity to an iCalendar event
+  def self.activity_to_event(activity, locale)
+    event = Icalendar::Event.new
+    event.uid = activity.id.to_s
+    event.dtstart = activity.start_date
+    event.dtend = activity.end_date
+    event.summary = activity.name
+    event.description = activity.description_localised(locale)
+    event.location = activity.location
+    return event
+  end
```

Comment:
I reckon the activity model class might be a better place for this method

Reply:
Agreed